rocAL/rocAL_pybind/examples/rocAL_api_pipeline.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
```

Comment:
@swetha097 i could see there are many file changes which are not necessarily related to audio
Please go over the PR once and removes the changes which are not required

Reply:
These changes are from the Python API PR#1
Once the Python API PR is merged, these changes would not be seen